pkg/compiler/compiler_use_test.go

```diff
@@ -0,0 +1,121 @@
+package compiler_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/MontFerret/ferret/pkg/compiler"
+	"github.com/MontFerret/ferret/pkg/stdlib/strings"
+	"github.com/MontFerret/ferret/pkg/stdlib/types"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUseExpression(t *testing.T) {
+	c := compiler.New()
+
+	c.Namespace("X").
+		RegisterFunction("INCLUDES", strings.Contains)
```

Comment:
Could you use a more unique name just to make sure that there is no a global function with such name?

Reply:
Smth like `XXX_CONTAINS`?

Reply:
Yes